### ISSUE TITLE
Add text card

### ISSUE THREAD

**Issue by timwis** [timwis/vizwit#113]
Just for text / paragraphs, most likely used to elaborate on adjacent cards

**Comment by jqnatividad** [comment]
+1. Would be nice if it supports Markdown.

**Comment by timwis** [comment]
Hey @jqnatividad! This would be pretty trivial to implement, though I've gone back and forth in my mind because of how easy it is to just [embed](http://vizw.it/embed-demo.html) the cards in your own layout. But I imagine this couldn't hurt. Are you interested in a CKAN implementation?

**Comment by jqnatividad** [comment]
Yes, very much so!.  Adding CKAN support is in our backlog as we see vizwit as something that we need to contribute to, similar to CitySDK when we contributed CKAN support.
